scripts/mk.js

```diff
@@ -3156,6 +3156,7 @@ function startGame() {
 									btnDir = 1;
 									break;
 								}
+								document.getElementById("game-container").style.filter = "brightness(100%)";
```

Comment:
```suggestion
								oContainers[0].style.filter = "brightness(100%)";
```

Reply:
Why did you add this line btw? It makes the effect disappear when I hit a key while in pause, not sure why we would want this :thinking:

Reply:
oh, oops. my bad 😅 I was about to sleep and never really considered stuff like this